scripts/issueEmailGenerator.js

```diff
@@ -84,6 +85,21 @@ if (typeof options.issueNumber !== 'number') {
     return `${process.env.NEXT_PUBLIC_ASSETS_URL}issue-${issueNumber}/${assetURL}`;
   }
 
+  function getAuthors(authors) {
+    if (authors?.length) {
+      console.log(authors);
+      const authorsWithWebsite = authors.map((author, index) => {
+        const isLastElement = index === authors.length - 1;
+        console.log(isLastElement);
+        return `[${author.Name}](${author.Website})` + (isLastElement ? '' : `, `);
```

Comment:
It can just be this -
```suggestion
        return `[${author.Name}](${author.Website})`;
```